macOS/DuckDuckGo/Subscription/SubscriptionManager+StandardConfiguration.swift

```diff
@@ -125,7 +125,13 @@ extension DefaultSubscriptionManagerV2 {
         let authClient = DefaultOAuthClient(tokensStorage: tokenStorage,
                                             legacyTokenStorage: legacyAccountStorage,
                                             authService: authService)
-        apiService.authorizationRefresherCallback = { _ in
+        apiService.authorizationRefresherCallback = { request in
+
+            guard request.url?.absoluteString.contains("api/auth/v2") == false else {
```

Comment:
this prevents attempts to re-authenticate an auth API request in the case of 401s and therefore will prevent authentication loops or useless retries